internal/tools/gen-genesis/main.go

```diff
@@ -98,9 +106,9 @@ func main() {
 		peer := fmt.Sprintf("%s@%s:26656", genVal.NodeID, genVal.Name)
 		logrus.WithFields(map[string]interface{}{
 			"name":     genVal.Name,
-			"address":  genVal.Address,
+			"address":  acc.GetAddress().String,
 			"password": genVal.Password,
-			"mnemonic": genVal.Mnemonic,
+			"mnemonic": mnemonic,
```

Comment:
you are generating the same account (identical mnemonic) just with different names. Each validator should have a different mnemonic, the same as the password

Reply:
No, it's not. There is a `for` line 78.

Reply:
my bad, I was confused by the passwords that are managed in a different loop